pipelines/lstm.py

```diff
@@ -0,0 +1,131 @@
+import os
+from tempfile import NamedTemporaryFile
+
+import polars as pl
+import tensorflow
+from keras import layers, losses, metrics, models, optimizers
+from keras.utils import timeseries_dataset_from_array
+from prefect import task
+from wandb.keras import WandbMetricsLogger
+
+import wandb
+from pipelines.types import TimeWindow
```

Comment:
Nitpick thought: is there a standard import ordering that Ruff could be configured to enforce?

Reply:
yea, one of the nice things about ruff, you can use all kinds of other tools in it as plugins, one common one is using isort, I can add this in